panels/move.py

```diff
@@ -13,6 +14,14 @@ class Panel(ScreenPanel):
 
     def __init__(self, screen, title):
         super().__init__(screen, title)
+
+        if self.ks_printer_cfg is not None:
+            dis = self.ks_printer_cfg.get("move_distances", '0.1, 0.5, 1, 50, 10, 25, 50')
+            if re.match(r'^[0-9,\s]+$', dis):
```

Comment:
the regular expression seems wrong, it doesn't work for floating point numbers (missing `\.` )

r'^[0-9,\.\s]+$'